src/components/ars.js

```diff
@@ -59,15 +61,17 @@ let Ars = createClass({
   },
 
   render() {
-    const { multiselect, resource } = this.props
+    const { multiselect, resource, rootAttributes } = this.props
     const { dialogOpen, picked } = this.state
 
     let SelectionComponent = multiselect ? MultiSelection : Selection
     let ref = multiselect ? 'multiselection' : 'selection'
     let slug = this.props.multiselect ? picked : picked && picked[0]
+    const rootClass = cx('ars', rootAttributes.className)
+    delete rootAttributes.className
 
     return (
-      <div className="ars">
+      <div className={rootClass} {...rootAttributes}>
```

Comment:
If you reverse the order of `className` and the `rootAttributes` spread, you can avoid the need to `delete rootAttributes.className`.

Reply:
👍